### ISSUE TITLE
custom selection mode won't work

### ISSUE THREAD

**Issue by akkos001** [Ultimaker/Cura#11610]
### Application Version

4.13.1

### Platform

Windows 10

### Printer

Artillery Sidewinder X1

### Reproduction steps

1. Opened "manage settings visibility"
2. Changed the visible settings for custom selection mode
3. Closed "manage settings visibility"
4. Changed to a different visibility mode (basic)
5. Clicked on custom selection

### Actual results

After I change back to the custom selection mode, the settings I had chosen get lost and it takes the settings visibility from the last mode I had. For example if I change from Basic to custom selection it will still show the basic settings visibility. Same goes for the other modes. The screenshots below show, that although I didn't change anything with the custom selection, it will just copy the mode I had activated before.
![Anmerkung 2022-03-05 103744](https://user-images.githubusercontent.com/99840477/156877742-51078c9f-6d1c-43e8-9271-aaed48449f70.png)
![Anmerkung 2022-03-05 103833](https://user-images.githubusercontent.com/99840477/156877745-45fc3fff-7761-4ede-aa53-d32df4952864.png)
![Anmerkung 2022-03-05 104026](https://user-images.githubusercontent.com/99840477/156877746-8399dfff-35f3-4e8d-b479-5a26e1939cdf.png)

### Expected results

It should show me the settings I have chosen before and not the preconfigured modes.

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

/

**Comment by fvrmr** [comment]
Hi @akkos001 thank you for your report.
I can't reproduce your issue. Could you share a screen capture?
![gif(43)](https://user-images.githubusercontent.com/77000924/157275812-5684bb99-82c4-4a8d-87fd-66baa939f59d.gif)

**Comment by akkos001** [comment]
Hello @fvrmr, thank you too for trying to find the problem. 

I believe this bug was caused due to some plug-ins I had installed.Before, I had added many plugins, which sounded useful. There Probably was a problem with a plugin. After I made this post, I uninstalled cura and reinstalled the clean version of cura without any plugins. I couldn't reproduce the bug. So I made a backup and slowly added the plugins which I need / use frequently, but the bug hasn't appeared yet. I will observate cura a little longer and if the bug won't happen again, I guess I can call this issue as fixed.
